### ISSUE TITLE
How to make custom dataset annotations?

### ISSUE THREAD

**Issue by kanadeqd** [stevewongv/SSIS#20]
Hi, thanks for your working. I'm confused about the **light** and **relation** of **annotations** and **association_anno**. 
I try to generate the SOBA annotations of by using #7 but the result are different from yours. 
![image](https://github.com/user-attachments/assets/5a8497ea-071f-4ce2-8834-8b2b78ecd5dc)
and the light of association_anno are also different from yours.
![image](https://github.com/user-attachments/assets/a0aa6e68-729e-45b2-afa7-41bcd1f1f7c7)

I noticed in datasets folder, there is no **prepare_thing_sem_from_instance.py**.
![image](https://github.com/user-attachments/assets/04906d88-ff59-44ec-892d-26078508d5b5)
Could you update this file and explain how to generate the annotations?

**Comment by WebsheetPlugin** [comment]
Did you solve this issue? @kanadeqd

**Comment by kanadeqd** [comment]
@WebsheetPlugin Yes, light is the center of image. relation is the center of shadow.
